src/lib.rs

```diff
@@ -1,31 +1,110 @@
-// Disable warning for the crate name, not a really good way to do this but..
-// (https://github.com/rust-lang/rust/issues/45127)
+// Disable warning for `non_snake_case` in the crate.
```

Comment:
Also worth mentioning that it disables that particular warning for the crates that use this one as dependency